tka/builder_test.go

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package tka
+
+import (
+	"testing"
```

Comment:
I think there should be some big test cases for multiple operations: both multiple of the same operation and a mix of different operations